README.md

```diff
@@ -21,7 +21,9 @@ Check the [releases page](https://github.com/IDPF/epubcheck/releases) to get the
 
 Documentation on how to **use** EpubCheck, to **contribute** to the project or to **translate** messages is available on the [EpubCheck wiki](https://github.com/IDPF/epubcheck/wiki).
 
-Technical discussions are hosted on the [EpubCheck Google Group](https://groups.google.com/forum/#!forum/epubcheck)
+Technical discussions are hosted are held on our public [email list](mailto:'public-epubcheck@w3.org'). To subscribe to the email list, send an email with subject "subscribe" to [public-epubcheck@w3.org](mailto:'public-epubcheck@w3.org'). To participate in the discussion, simply send an email to [public-epubcheck@w3.org](mailto:'public-epubcheck@w3.org'). 
```

Comment:
@iherman would you please confirm?

Reply:
oh gosh, you're totally right!

And I also screwed this up in my mail to the Google Group list 😱  Damn!

Reply:
> And I also screwed this up in my mail to the Google Group list 😱 Damn!

@iherman Ivan, does the new list already has some subscribers or did I really screw it up by providing all the old users with wrong signup instructions? 😟

Reply:
I tested to see if you'd get a message back with instructions on how to join the list, but all the message returns is a link to allow the message to be archived before it will be posted. If people do that we will get subscribe messages to the list.

It's not the end of the world, though. So long as we get it right for the repository it'll still be easy enough for people to find the new list.

Reply:
@mattgarrish thanks for testing this. we didn't get any messages on the list yet, which means, none of the 190 subscribers of the old list tried to sign up yet :-(

Shall we re-open the list for a sec and send corrected signup instructions? Would be worth a try....

What do you others think?